spec/ar2dto/has_dto_spec.rb

```diff
@@ -115,6 +118,10 @@
 
         expect(subject).not_to eq(admin)
       end
+
+      it "is not possible to set values from outside" do
```

Comment:
Ahh, so the change about the attr writer to be private was for this?

If I understood correctly, I'm not sure if this restriction to the gem usage is actually something good to have